js-intro.js

```diff
@@ -23,8 +38,29 @@ var myCat = "Garfield"
 
 // 2a. Create a function that returns the longer of the two variables. Expected output: "Garfield"
 
+// var myDog = "Snoopy"
+// var myCat = "Garfield"
+// function dogCat (str1, str2){
+//     if (str1.length>str2.length){
+//          console.log(str1)
+//         return str1 
+//     }else{
+//         console.log(str2)
+//         return str2
+    
+//     }
+// }
+//     dogCat(myDog, myCat)
+
```

Comment:
You can shorten this up a bit by console logging the function call rather than logging and returning each of your conditional statements.